src/weakptr.c

```diff
@@ -194,16 +194,14 @@ static inline void GROW_WPOBJ(Obj wp, UInt need)
     if (plen > INT_INTOBJ_MAX)
         plen = INT_INTOBJ_MAX;
 
-#ifdef USE_BOEHM_GC
```

Comment:
Tthis bit of the PR seems unrelated, did it get merged in accidentally? I also wasn't sure if it was reasonable (I've not used Boehm in a long time, to know if this is correct).